Tests/DatadogTests/Datadog/RUMMonitorTests.swift

```diff
@@ -959,60 +966,12 @@ class RUMMonitorTests: XCTestCase {
         }
     }
 
-    // MARK: - Tracking Consent
-
-    func testWhenChangingConsentValues_itUploadsOnlyAuthorizedRUMEvents() throws {
-        let consentProvider = ConsentProvider(initialConsent: .pending)
-
```

Comment:
Same as in [logging PR](https://github.com/DataDog/dd-sdk-ios/pull/877#discussion_r887716663) - consent aware uploads a no longer responsibility of Feature so we can't test it here.